app/models/account.rb

```diff
@@ -540,6 +544,13 @@ def journal_entry_items_calculate(column, started_at, stopped_at, operation = :s
     journal_entry_items.where(printed_on: started_at..stopped_at).calculate(operation, column)
   end
 
+  def is_number_valid?
+    is_valid = true
+    split_number = self.number.split("")
+    is_valid = false if split_number.first == "0" || split_number.drop(1).all? { |n| n == "0" }
```

Comment:
I don't see how this prevents from writing in the radix accounts (1, 2, 3, 4, 5, 6, 7, 8) ?